.travis.yml

```diff
@@ -2,6 +2,7 @@
 # new container-based Travis workers
 language: python
 dist: xenial
+python: 3.9
```

Comment:
This is only used to install cibuildwheel, `CIBW_BUILD` determines the python version being built in the docker image.

(Also Travis CI does not have 3.10 yet)

Reply:
```suggestion
# Only used to install cibuildwheel, CIBW_BUILD determines the python version being
# built in the docker image itself. Also: travis does not have 3.10 yet.
python: 3.9
```